app/models/knight.rb

```diff
@@ -1,2 +1,19 @@
 class Knight < Piece
+  def obstructed_diagonally?
```

Comment:
Oh yeah, I see that now. Maybe we could try passing it `(*)` as rubocop suggests. Reason being is that we can't test the over-ride without calling the method.